sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -132,15 +134,133 @@ export interface ServiceBusReceiver<ReceivedMessageT> {
    * Use the `createReceiver()` method on the ServiceBusClient to create a new Receiver.
    */
   close(): Promise<void>;
+  /**
+   * Removes the message from Service Bus.
+   *
+   * @throws Error with name `SessionLockLostError` (for messages from a Queue/Subscription with sessions enabled)
+   * if the AMQP link with which the message was received is no longer alive. This can
+   * happen either because the lock on the session expired or the receiver was explicitly closed by
+   * the user or the AMQP link got closed by the library due to network loss or service error.
+   * @throws Error with name `MessageLockLostError` (for messages from a Queue/Subscription with sessions not enabled)
+   * if the lock on the message has expired or the AMQP link with which the message was received is
+   * no longer alive. The latter can happen if the receiver was explicitly closed by the user or the
+   * AMQP link got closed by the library due to network loss or service error.
+   * @throws Error if the message is already settled. To avoid this error check the `isSettled`
+   * property on the message if you are not sure whether the message is settled.
+   * @throws Error if used in `ReceiveAndDelete` mode because all messages received in this mode
```

Comment:
```suggestion
   * @throws Error if used in `receiveAndDelete` mode because all messages received in this mode
```

Reply:
Same at other places in this file

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11962/commits/3ec99efacab643ad6fa9ddae7c225ebbbb7231b2